<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="XIPL0639" name="Surefire test" tests="13" failures="6" timestamp="13 Aug 2019 08:58:54 GMT" time="0.027" errors="0">
  <testcase name="test1" time="0.008" classname="com.global.test.GlobTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.global.test.GlobTest.test1(GlobTest.java:23)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase name="test10" time="0.0" classname="com.global.test.GlobTest">
    <skipped/>
  </testcase> <!-- test10 -->
  <testcase name="test2" time="0.0" classname="com.global.test.GlobTest">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test3" time="0.0" classname="com.global.test.GlobTest">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase name="test4" time="0.0" classname="com.global.test.GlobTest">
    <failure type="java.lang.AssertionError" message="expected [10] but found [20]">
      <![CDATA[java.lang.AssertionError: expected [10] but found [20]
	at com.global.test.GlobTest.test4(GlobTest.java:44)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test4 -->
  <testcase name="test5" time="0.0" classname="com.global.test.GlobTest">
    <failure type="java.lang.AssertionError" message="expected [https://www.amazon.com/] but found [Amazon]">
      <![CDATA[java.lang.AssertionError: expected [https://www.amazon.com/] but found [Amazon]
	at com.global.test.GlobTest.test5(GlobTest.java:49)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
  <testcase name="test6" time="0.0" classname="com.global.test.GlobTest">
    <failure type="java.lang.AssertionError" message="expected [edgeop] but found [edgeops]">
      <![CDATA[java.lang.AssertionError: expected [edgeop] but found [edgeops]
	at com.global.test.GlobTest.test6(GlobTest.java:54)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test6 -->
  <testcase name="test7" time="0.0" classname="com.global.test.GlobTest"/>
  <testcase name="test8" time="0.0" classname="com.global.test.GlobTest"/>
  <testcase name="test9" time="0.0" classname="com.global.test.GlobTest">
    <skipped/>
  </testcase> <!-- test9 -->
  <testcase name="test_Title" time="0.0" classname="com.global.test.GlobTest">
    <failure type="java.lang.AssertionError" message="expected [ola] but found [uber]">
      <![CDATA[java.lang.AssertionError: expected [ola] but found [uber]
	at com.global.test.GlobTest.test_Title(GlobTest.java:18)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test_Title -->
  <testcase name="test_Url" time="0.0" classname="com.global.test.GlobTest">
    <failure type="java.lang.AssertionError" message="expected [https://www.olx.com/] but found [olx]">
      <![CDATA[java.lang.AssertionError: expected [https://www.olx.com/] but found [olx]
	at com.global.test.GlobTest.test_Url(GlobTest.java:11)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- test_Url -->
  <testcase name="test_serachText" time="0.0" classname="com.global.test.GlobTest"/>
</testsuite> <!-- Surefire test -->
